components/automate-ui/src/app/pages/team-details/team-details.component.ts

```diff
@@ -233,29 +218,6 @@ export class TeamDetailsComponent implements OnInit, OnDestroy {
             }
           });
       });
-
-    // we keep a list of all projects that were left in the team
-    // to know if we are fully loaded yet or not.
-    combineLatest([
-      this.store.select(getProjectStatus),
-      this.store.select(projectEntities)])
-      .pipe(filter(([status, _]: [EntityStatus, ProjectCheckedMap]) =>
-          status === EntityStatus.loadingSuccess),
```

Comment:
This was actually a race condition, of sorts, too. We sent off requests for several projects--but they all use the same slot in the store for setting their returned status. Thus, we got someone's success, but don't know for whom!